test/warp.jl

```diff
@@ -427,15 +427,26 @@ ref_img_pyramid_grid = Float64[
                 @test_nowarn imrotate(img, π/4, Linear())
                 @test_nowarn imrotate(img, π/4, axes(img))
                 @test_nowarn imrotate(img, π/4, axes(img), Constant())
-                @test isequal(channelview(imrotate(img,π/4)), channelview(imrotate(img, π/4, Linear()))) # TODO: if we remove channelview the test will break for Float
+                @test isequal(channelview(imrotate(img, π/4)), channelview(imrotate(img, π/4, Linear()))) # NaN != NaN
```

Comment:
The meaning of this comment is unclear. Do you mean that you wanted to write `==` but instead have to use `isequal`?